app/views/drawings/_drawing.html.haml

```diff
@@ -55,4 +55,14 @@
             .drawing-row
               %strong Context / Story:
               = drawing.story
-            
+            %br
+            .drawing-row
+              - case drawing.image_consent
+              - when nil
+                %span.fa.fa-question
```

Comment:
Think we can avoid this case altogether by implementing the suggestion below re not allowing nulls for this field (i.e. default to false in the migration, still can default the UI to true)